data/images/engine/editor/objects.stoi

```diff
@@ -297,6 +297,9 @@
   (object
     (class "skull_tile")
     (icon "images/objects/skull_tile/skull.png"))
+  (object
+    (class "circleplatform")
+    (icon "images/objects/platforms/icebridge1.png"))
```

Comment:
Please, could you make dedicated icon (potentialy by modifying bicycle-platform one)?